supabase/migrations/2023101016_remote_schema.sql

```diff
@@ -0,0 +1,317 @@
+CREATE EXTENSION IF NOT EXISTS "pg_stat_statements" WITH SCHEMA "extensions";
+CREATE EXTENSION IF NOT EXISTS "pgcrypto" WITH SCHEMA "extensions";
+CREATE EXTENSION IF NOT EXISTS "pgjwt" WITH SCHEMA "extensions";
+CREATE EXTENSION IF NOT EXISTS "supabase_vault" WITH SCHEMA "vault";
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp" WITH SCHEMA "extensions";
+
+SET default_tablespace = '';
+SET default_table_access_method = "heap";
+
+-- Existing tables (unchanged)
+CREATE TABLE IF NOT EXISTS "public"."credits" (
+    "id" bigint NOT NULL,
+    "created_at" timestamp with time zone DEFAULT now() NOT NULL,
+    "credits" integer DEFAULT 0 NOT NULL,
+    "user_id" uuid NOT NULL
+);
+
+ALTER TABLE "public"."credits" OWNER TO "postgres";
+
+ALTER TABLE "public"."credits" ALTER COLUMN "id" ADD GENERATED BY DEFAULT AS IDENTITY (
+    SEQUENCE NAME "public"."credits_id_seq"
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1
+);
+
+CREATE TABLE IF NOT EXISTS "public"."images" (
+    "id" bigint NOT NULL,
+    "modelId" bigint NOT NULL,
+    "uri" text NOT NULL,
+    "created_at" timestamp with time zone DEFAULT now() NOT NULL
+);
+
+ALTER TABLE "public"."images" OWNER TO "postgres";
+
+ALTER TABLE "public"."images" ALTER COLUMN "id" ADD GENERATED BY DEFAULT AS IDENTITY (
+    SEQUENCE NAME "public"."images_id_seq"
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1
+);
+
+CREATE TABLE IF NOT EXISTS "public"."models" (
+    "id" bigint NOT NULL,
+    "name" text,
+    "type" text,
+    "created_at" timestamp with time zone DEFAULT now() NOT NULL,
+    "user_id" uuid,
+    "status" text DEFAULT 'processing'::text NOT NULL,
+    "modelId" text
+);
+
+ALTER TABLE "public"."models" OWNER TO "postgres";
+
+ALTER TABLE "public"."models" ALTER COLUMN "id" ADD GENERATED BY DEFAULT AS IDENTITY (
+    SEQUENCE NAME "public"."models_id_seq"
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1
+);
+
+CREATE TABLE IF NOT EXISTS "public"."samples" (
+    "id" bigint NOT NULL,
+    "uri" text NOT NULL,
+    "modelId" bigint NOT NULL,
+    "created_at" timestamp with time zone DEFAULT now() NOT NULL
+);
+
+ALTER TABLE "public"."samples" OWNER TO "postgres";
+
+ALTER TABLE "public"."samples" ALTER COLUMN "id" ADD GENERATED BY DEFAULT AS IDENTITY (
+    SEQUENCE NAME "public"."samples_id_seq"
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1
+);
+
+-- New tables
+CREATE TABLE IF NOT EXISTS "public"."users" (
+    "id" uuid PRIMARY KEY,
+    "email" text,
+    "name" text,
+    "credit_balance" integer,
+    "last_credit_update" timestamp with time zone,
+    "created_at" timestamp with time zone DEFAULT now(),
+    "updated_at" timestamp with time zone DEFAULT now(),
+    "last_login_at" timestamp with time zone
+);
+
+CREATE TABLE IF NOT EXISTS "public"."plans" (
+    "id" bigint PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
```

Comment:
check if you can directly add the start_value and increment_by so that you need not have the ALTER command running on every table